CHANGELOG.md

```diff
@@ -47,7 +48,7 @@
   - `bround`.
   - `substring_index`
 - Added parameter `skip_upload_on_content_match` when creating UDFs, UDTFs and stored procedures using `register_from_file` to skip uploading files to a stage if the same version of the files are already on the stage.
-- Added support for `DataFrame.save_as_table` method to take table names that contain dots.
+- Added support for `DataFrameWriter.save_as_table` method to take table names that contain dots.
```

Comment:
I'm not sure if we should correct old changelog 🤔

Reply:
I lean towards to correcting it, so that people won't be confused when they find no `save_as_table` on `DataFrame` when they check our changelog history. this will show up in the default pypi home page and our github main branch